astropy/cosmology/parameter/_core.py

```diff
@@ -70,6 +83,9 @@ class Parameter:
 
     Parameters
     ----------
+    default : Any (optional, keyword-only)
+        Default value of the Parameter. If ``MISSING`` (default), the
```

Comment:
I would write "If not given, the" - no need to mention the sentinel value here.

Reply:
Done.